.github/workflows/portal-loop.yml

```diff
@@ -1,13 +1,14 @@
 name: portal-loop
 
 on:
+  pull_request: {}
   push:
     paths:
       - misc/loop
       - .github/workflows/portal-loop.yml
     branches:
       - "master"
-      - "ops/portal-loop"
+      - "ci/portal-loop"
```

Comment:
add a comment explaining this branch, please.

Reply:
done